jplag.endToEndTesting/src/main/java/de/jplag/endToEndTesting/helper/JsonReader.java

```diff
@@ -0,0 +1,53 @@
+package de.jplag.endToEndTesting.helper;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import java.io.Reader;
+
+import java.nio.charset.Charset;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class JsonReader {
+
+	private static String readAll(Reader rd) throws IOException {
+		StringBuilder sb = new StringBuilder();
+		int cp;
+		while ((cp = rd.read()) != -1) {
+			sb.append((char) cp);
+		}
+		return sb.toString();
+	}
```

Comment:
new Scanner(inputStream).useDelimiter("\\A").next()

Reply:
Was removed by the change with the jsonParser. 
``` java 
public static List<ResultJsonModel> getResultModelFromPath(String pathToJsonFile)
			throws JsonMappingException, JsonProcessingException, Exception {
		ObjectMapper objectMapper = new ObjectMapper();
                return Arrays.asList(objectMapper.readValue(Paths.get(pathToJsonFile).toFile(), ResultJsonModel[].class));
	}
```